### PR TITLE
Cyberprodigy hometown

### PR DESCRIPTION
**Pull request by andersrosbaek**
# This could go horribly wrong.

### CODE REVIEW

**Comment by andersrosbaek** [comment]
Pull requesties?

File: cyberprodigy-gaujiena.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+## Gaujiena, LV
+
+### Places to eat
+- Melderitis
+Party rooms (60 persons), meals, party service, caf�, volleyball grounds, boating down the Gauja river.
+
+
+### Fun things to do
+- Play some sports
```
Comment (emindevekaya) [inline cyberprodigy-gaujiena.md]: Careful, don't break your leg!

Comment (emindevekaya) [inline cyberprodigy-gaujiena.md]: Pow!

File: cyberprodigy-gaujiena.md [hunk 2]
```diff
@@ -6,9 +6,4 @@ Party rooms (60 persons), meals, party service, caf
...
 
 ### Fun things to do
-- Play some sports
-
-
-<script>
-alert("");
-</script>
\ No newline at end of file
+- Play some sports
```
Comment (emindevekaya) [inline cyberprodigy-gaujiena.md]: Ughh

File: cyberprodigy-gaujiena.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+## Gaujiena, LV
```
Comment (emindevekaya) [inline cyberprodigy-gaujiena.md]: Nice place

**Comment by andersrosbaek** [comment]
Wooop!

**Pull request merged**